pkg/security/tests/latency/Makefile

```diff
@@ -0,0 +1,15 @@
+name = bin/bench_net_DNS
```

Comment:
Do we still need this Makefile with the `build-latency-tools` task ?

Reply:
Nope, we can get rid of it